### ISSUE TITLE
supporting the new "Policy fragments" feature in the extractor

### ISSUE THREAD

**Issue by liorklein** [Azure/azure-api-management-devops-resource-kit#736]
Do you have roadmap for adding support for the new feature of apim  "Policy fragments" to the extractor

**Comment by f-alizada** [comment]
Hello @liorklein.
The issue was closed since the PR introducing exporting policy fragments functionality is merged (#775).
Please feel free to report a bug in case you will face some issues with this. 
Best regards.

**Comment by leonardosantoss** [comment]
Hi @f-alizada ,

I have encountered a behaviour which I am not sure if it is expected or not.

I have been only able to generate the policy fragments template if I extract all api's. 

For example I have an API "api_example". And a policy fragment with the following reference: api_example;rev=1/api_endpoint

Extractor Params that works:

`{

    "sourceApimName": "apim-name",
    "destinationApimName": "apim-name",
    "resourceGroup": "rg-name",
    "fileFolder": "./src",
    "paramNamedValue": "true",
    "paramServiceUrl": "true",
    "paramNamedValuesKeyVaultSecrets": "true",
    "splitAPIs": "true",
    "linkedTemplatesSasToken": "needsOverride",
    "policyXMLSasToken": "needsOverride",
    "linkedTemplatesBaseUrl": template-url,
    "policyXMLBaseUrl": template-url
}

 `
When extracting all api's, the following relevant files are generated:

- apim-name-policy-fragments.template.json
- api_example/api_example;rev=1/apim-name-policy-fragments.template.json
- api_example/api_example;rev=2/apim-name-policy-fragments.template.json
- api_example/VersionSetMasterFolder/apim-name-policy-fragments.template.json
- api-name-master.template.json contains reference to policy fragments


Extractor Params that do not work:

`
{
  "apiName": "api_example",
  "sourceApimName": "apim-name",
  "destinationApimName": "apim-name",
  "resourceGroup": "rg-name",
  "fileFolder": "./src",
  "paramNamedValue": "true",
  "paramServiceUrl": "true",
  "paramNamedValuesKeyVaultSecrets": "true",
  "linkedTemplatesSasToken": "needsOverride",
  "policyXMLSasToken": "needsOverride",
  "linkedTemplatesBaseUrl": endpoint-url/templates/,
  "policyXMLBaseUrl": endpoint-url/policies/
}
`
I also tried with "apiName" as "api_example;rev=1" and also adding the field includeAllRevisions to true.

Are policy fragments supposed to be generated in those cases? Thank you for your help!
